scripts/deploy_oak_functions_api

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+
+readonly SCRIPTS_DIR="$(dirname "$0")"
+# shellcheck source=scripts/common
+source "$SCRIPTS_DIR/common"
+
+# shellcheck source=scripts/gcp_common
+source "$SCRIPTS_DIR/gcp_common"
+
+gcloud auth activate-service-account \
+  --project="${GCP_PROJECT_ID}" \
+  --key-file="${GCP_ACCOUNT_FILE}"
+
+# Rebuild the proto descriptor file for the API configuration.
+protoc \
+  --proto_path=./oak_functions/proto \
+  --include_imports \
+  --include_source_info \
+  --descriptor_set_out=./oak_functions/gcp/server.pb \
```

Comment:
Why is `oak_functions/gcp/server.pb` checked-in even though it is generated here?

Reply:
It is reusable for manual deployments as well, so I thought it would be a useful reference point to check it in.